incubator/prometheus/alertmanager.yml

```diff
@@ -0,0 +1,14 @@
+global:
+  slack_api_url: '...'
```

Comment:
what happens if this default isnt changed? Will things still work as expected? Can we comment it out and add a describe what should be set here?